docs/source/notebooks/wps_generate_climos_demo.ipynb

```diff
@@ -0,0 +1,403 @@
+    "    operation=operation, \n",
+    "    climo=climo, \n",
+    "    resolutions=resolutions, \n",
+    "    dry_run=dry_run\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<metaurl mediatype=\"text/plain\">\n",
+      "     https://docker-dev03.pcic.uvic.ca/wpsoutputs/bf506e92-041c-11eb-8e5f-0242ac120004/fdd_seasonal_CanESM2_rcp85_r1i1p1_1951-2100_dry.txt\n",
+      "    </metaurl>\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Process dry run output\n",
+    "req = requests.get(dry_output.get()[0])\n",
+    "metalink = BeautifulSoup(BeautifulSoup(req.content.decode('utf-8')).prettify()).metaurl\n",
+    "print(metalink)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Test dry run output is as expected\n",
+    "url = metalink.get_text()\n",
+    "with urlopen(url) as f:\n",
+    "    output_data = f.read().decode('utf-8')\n",
+    "\n",
+    "expected_items = ['6190', 'CMIP5', 'PCIC', 'CanESM2', 'historical', 'rcp85', 'r1i1p1', 'fdd', 'seasonal']\n",
+    "for item in expected_items:\n",
+    "    assert item in output_data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Normal Run**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# generate climos\n",
+    "output = thunderbird.generate_climos(\n",
+    "    netcdf=seasonal_opendap, \n",
+    "    operation=operation, \n",
+    "    climo=climo, \n",
+    "    resolutions=resolutions, \n",
+    "    dry_run=False\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<metaurl mediatype=\"application/x-netcdf\">\n",
+      "     https://docker-dev03.pcic.uvic.ca/wpsoutputs/c2439214-041c-11eb-8e5f-0242ac120004/fdd_aClimMean_BCCAQ_CanESM2_historical+rcp85_r1i1p1_19610101-19901231_Canada.nc\n",
+      "    </metaurl>\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Process normal output\n",
+    "req = requests.get(output.get()[0])\n",
+    "metalink = BeautifulSoup(BeautifulSoup(req.content.decode('utf-8')).prettify()).metaurl\n",
+    "print(metalink)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Helper function to test netCDF file output\n",
+    "def test_nc_data(url):\n",
+    "    with NamedTemporaryFile(suffix=\".nc\", prefix=\"tmp_copy\", dir=\"/tmp\", delete=True) as tmp_file:\n",
+    "        urlretrieve(url, tmp_file.name)\n",
+    "        output_data = Dataset(tmp_file.name)\n",
+    "\n",
+    "        assert output_data.dimensions['time'].size == 1"
```

Comment:
It would be nice if there is a docstring that explains what the function is testing/checking.

Reply:
Should I do a doc strong or just markdown?

Reply:
docstring would be fine